src/amo/components/AddonRecommendations/index.js

```diff
@@ -114,9 +114,9 @@ export class AddonRecommendationsBase extends React.Component<Props> {
   }
 
   render() {
-    const { className, i18n, recommendations } = this.props;
+    const { className, i18n, recommendations, errorHandler } = this.props;
 
-    if (!recommendations) {
+    if (!recommendations || errorHandler.hasError()) {
       log.debug(
         'No recommendations, hiding the AddonRecommendations component.',
```

Comment:
It might be worth outputting a different message if there is an error. In that case it's not really because there are no recommendations, so the message could be misleading.